pkg/kubectl/cmd/drain.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/jonboulle/clockwork"
 	"github.com/spf13/cobra"
 
+	corev1 "k8s.io/api/core/v1"
```

Comment:
do we already have that dependency? (I don't see a corresponding bazel BUILD file change)